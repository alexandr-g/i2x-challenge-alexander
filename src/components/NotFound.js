import React from 'react';
import { Link } from 'react-router';

const NotFound = () => {
  return (
    <div className="ph3 mt4">
      <h1 className="f6 fw6 ttu tracked">Page not found.</h1>
      <Link to="/" className="f6 grow no-underline br-pill ba bw2 ph3 pv2 mb2 dib black">Home</Link>
    </div>
  );
};

export default NotFound;
