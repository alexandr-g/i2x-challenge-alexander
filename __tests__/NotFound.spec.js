/* env jest */

import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../src/components/NotFound';

describe('NotFound page', () => {
  test('is rendered', () => {
    const component = renderer.create(<NotFound />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
