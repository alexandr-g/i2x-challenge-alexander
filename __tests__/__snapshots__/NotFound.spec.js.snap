exports[`NotFound page is rendered 1`] = `
<div
  className="ph3 mt4">
  <h1
    className="f6 fw6 ttu tracked">
    Page not found.
  </h1>
  <a
    className="f6 grow no-underline br-pill ba bw2 ph3 pv2 mb2 dib black"
    onClick={[Function]}
    style={Object {}}>
    Home
  </a>
</div>
`;
